app/src/test/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentLogicTest.kt

```diff
@@ -549,41 +573,39 @@ class ScheduleAppointmentSheetControllerTest {
 
     //when
     sheetCreated()
-    uiEvents.onNext(PatientFacilityChanged(updatedFacilityUuid))
+    uiEvents.onNext(PatientFacilityChanged(updatedFacility))
 
     //then
     val inOrder = inOrder(ui)
     inOrder.verify(ui).showPatientFacility(facility.name)
     inOrder.verify(ui).showPatientFacility(updatedFacility.name)
+
+    verifyZeroInteractions(uiActions)
   }
 
   private fun sheetCreated(
-      patientUuid: UUID = this.patientUuid,
       facility: Facility = this.facility,
       protocol: Protocol = this.protocol,
       config: AppointmentConfig = this.appointmentConfig
   ) {
     setupMobiusTestFixture(facility, config)
-    setupController(patientUuid, config, facility)
 
     whenever(protocolRepository.protocol(protocol.uuid)).thenReturn(Observable.just(protocol))
     whenever(protocolRepository.protocolImmediate(protocol.uuid)).thenReturn(protocol)
 
-    activateUi()
```

Comment:
`activateUi` sounds like we are starting a robot 😛

Reply:
🤖

Reply:
Good that it got removed then, I do not want to be responsible for Skynet.